Code/autopkglib/__init__.py

```diff
@@ -420,7 +429,7 @@ def get_recipe_identifier(self, recipe):
         """Return the identifier given an input recipe plist."""
         identifier = recipe.get("Identifier") or recipe["Input"].get("IDENTIFIER")
         if not identifier:
-            print("ID NOT FOUND")
+            log("ID NOT FOUND")
```

Comment:
Although this was originally to stdout, this seems like it should logically go to stderr.

Reply:
Ok. 988df72

Reply:
I think the point is more that since you are no longer calling `print()`, having replaced all those calls with `log()`, you no longer need to `from __future__ import print_function`, and if you leave that in, the various linters might complain about unused imports.